shell/probe/raindrop.ps1

```diff
@@ -32,7 +42,7 @@ function FromEnv { param ([string]$envVar, [string]$default)
 }
 
 $ca = FromEnv "PRELUDE_CA" "prelude-account-us1-us-east-2.s3.amazonaws.com"
-$dir = FromEnv "PRELUDE_DIR" $ca
+$dir = FromEnv "PRELUDE_DIR" ".vst"
```

Comment:
the other diff is that nocturnal doesn't even check for an env variable PRELUDE_DIR. do we care about aligning there too?